src/api-client.ts

```diff
@@ -57,9 +99,41 @@ export class RevAiApiClient {
             { 'Content-Type': 'application/json' }, 'json', options);
     }
 
-    async submitJobLocalFile(filename: string, options?: RevAiJobOptions): Promise<RevAiApiJob> {
+    /**
+     * See https://www.rev.ai/docs#operation/SubmitTranscriptionJob
+     * Submit audio data for transcription.
+     * @param fileStream Audio data to be transcribed.
+     * @param filename Name of file associated with audio, file extension is used to determine type of audio.
```

Comment:
maybe make this optional? I would think from `buffer`, there isn't always a `filename` present.

Uncertain what's the best experience right now, I just think it's kind of confusing to have to pass a filename when the name of the method seems to indicate all we need is a buffer of data

Reply:
filename needs to be provided otherwise the API won't accept the "media file" since it has no name or (more importantly) content type

Reply:
If the filename is not provided by the user, we can generate it ourselves

Reply:
I guess the content-type can be tricky

Reply:
I originally thought content type was coming from filename. It is not, we can make this optional and just use a generic, extensionless filename